src/Tribe/Views/V2/Hooks.php

```diff
@@ -635,4 +638,45 @@ public function filter_get_stylesheet_option( $value, $key ) {
 
 		return $value;
 	}
+
+	/**
+	 * Filter the liveFiltersUpdate option to do some switching for V2.
+	 * Note: this triggers on option_liveFiltersUpdate, tribe_get_option, AND tribe_field_value. We
+	 * don't have to add/remove filters because we don't need to get the value - it's already provided.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $value      The option value.
+	 * @param string $optionName The option key.
+	 *
+	 * @return void
```

Comment:
:no_entry_sign: Function return type is void, but function contains return statement

<sub>hash: 97503e4dd9c1ceb0cc9d8820e7284ed5</sub>